packages/main/src/MessageViewItem.ts

```diff
@@ -0,0 +1,105 @@
+import UI5Element from "@ui5/webcomponents-base/dist/UI5Element.js";
+import customElement from "@ui5/webcomponents-base/dist/decorators/customElement.js";
+import property from "@ui5/webcomponents-base/dist/decorators/property.js";
+import event from "@ui5/webcomponents-base/dist/decorators/event.js";
+import litRender from "@ui5/webcomponents-base/dist/renderer/LitRenderer.js";
+
+// Styles
+
+import { MessageViewMode } from "./MessageView.js";
+
+/**
+ * @class
+ *
+ * <h3 class="comment-api-title">Overview</h3>
+ *
+ * The <code>ui5-message-view-item/code> component is used to display a single message with status, title, subtitle and description
+ *
+ * <h3>Usage</h3>
+ *
+ * For the <code>message-view-item</code>
+ * <h3>ES6 Module Import</h3>
+ *
+ * <code>import @ui5/webcomponents/dist/MessageViewItem.js";</code>
+ *
+ * @constructor
+ * @author SAP SE
+ * @alias sap.ui.webc.main.MessageViewItem
+ * @extends sap.ui.webc.base.UI5Element
+ * @tagname message-view-item
+ * @public
+ */
+@customElement({
+	tag: "ui5-message-view-item",
+	renderer: litRender,
+	themeAware: true,
```

Comment:
remove themeAware/languageAware since we don't have template which needds to be rerendered.

Reply:
I left only the tag name and removed the renderer and theme and language awareness